.github/workflows/deploy.yml

```diff
@@ -0,0 +1,29 @@
+name: deploy
```

Comment:
GitHub is showing this as a new file in the diff, but it's really a renamed and modified version of `main.yml`.